server/src/workload_state_db.rs

```diff
@@ -15,8 +15,11 @@
 use common::objects::{ExecutionState, WorkloadState};
 use std::collections::HashMap;
 
-type WorkloadStatesMap = HashMap<String, common::objects::ExecutionState>;
-type AgentWorkloadStates = HashMap<String, WorkloadStatesMap>;
+type WorkloadName = String;
+type AgentName = String;
+
+type WorkloadStatesMap = HashMap<WorkloadName, Vec<WorkloadState>>;
+type AgentWorkloadStates = HashMap<AgentName, WorkloadStatesMap>;
 
 pub struct WorkloadStateDB {
```

Comment:
For this module only the insert method is linked. No utest or other public method is linked at all.

Reply:
Done.